x-pack/plugins/integration_assistant/server/util/samples.ts

```diff
@@ -233,3 +233,45 @@ export function mergeSamples(objects: any[]): string {
 
   return JSON.stringify(result, null, 2);
 }
+
+export function flattenObjectsList(
+  obj: any[]
```

Comment:
We expect some specific structure for `obj` (is it the `Field` interface?), so we should document it in the doc string, especially if we keep `any[]` as a type.

Reply:
we can use `Field`, I have updated the param type. I had to export the interface to be able to reference it outside of the `sample` module. Let me know if there is a better place to define it now that it is not specific to the `sample` module only

Reply:
This looks good to me.